portal/portal_payment.php

```diff
@@ -1285,7 +1285,7 @@ function getAuth() {
         if (!isset($_SESSION['authUserID'])) {
             if (!isset($ccdata["cardHolderName"])) {
                 if ($GLOBALS['payment_gateway'] == 'Sphere') {
-                    echo SpherePayment::renderSphereHtml('patient');
+                    echo SpherePayment::renderSphereHtml();
```

Comment:
gotta revert this line back :)

Reply:
Yep, continue to keep an eye on me @bradymiller and thanks...